pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -932,7 +932,7 @@
     "zeroconf" = ps: with ps; [ aiohttp-cors zeroconf];
     "zerproc" = ps: with ps; [ ]; # missing inputs: pyzerproc
     "zestimate" = ps: with ps; [ xmltodict];
-    "zha" = ps: with ps; [ pyserial zha-quirks zigpy-deconz]; # missing inputs: bellows zigpy-cc zigpy-xbee zigpy-zigate zigpy
+    "zha" = ps: with ps; [ bellows pyserial zha-quirks zigpy zigpy-cc zigpy-deconz zigpy-xbee zigpy-zigate];
```

Comment:
Note of interest which I don't know what to do about:

When I ran the generator to update this file it did put the `zigpy` dependency at the end of the list. Then home-assistant failed to load `zigpy.config` because something seemed to depend on it before it was loaded. This may be an upstream issue in how the requirements list is parsed, otherwise it's a sorting issue in our updater file.

cc @FRidh